servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/primitives/ReceiverErrorContext.java

```diff
@@ -7,6 +7,8 @@
 
 public class ReceiverErrorContext extends ErrorContext
 {
+    private static final long serialVersionUID = -8154706630781986787L;
```

Comment:
[SpotBugs]
Since ServiceBusException has serialVersionUID, it means it needs to implement Serializable.  Change the ErrorContext to implements Serializable cause all its sub-extended class to have the serialVersionUID